.github/workflows/deploy_ecs_TESTNET.yml

```diff
@@ -11,14 +11,15 @@ env:
   PROJECT_NAME: "relayers"
   TAG: "Demo"
   AWS_TESTNET: '${{ secrets.AWS_ARN }}'
+  AWS_ROLE: '${{ secrets.AWS_ROLE_NAME }}'
 
 jobs:
   deploy:
     name: deploy
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        relayer_id: [0, 1, 2]
+        relayer_id: [0]
```

Comment:
Why was this changed?

Reply:
By default I want to deploy only one relayer and add description how to increase amount of relayers later.

Reply:
Just realized I was looking at the wrong repo and thought these were our current testnet task files.